src/panels/config/integrations/integration-panels/zwave_js/zwave_js-node-config.ts

```diff
@@ -319,8 +321,32 @@ class ZWaveJSNodeConfig extends LitElement {
         </div>
       `;
     }
-
     if (item.configuration_value_type === "manual_entry") {
+      if (
+        item.metadata.states &&
+        item.metadata.min != null &&
+        item.metadata.max != null &&
+        item.metadata.max - item.metadata.min <= 100
+      ) {
+        return html`
+          ${labelAndDescription}
+          <ha-combo-box
+            .hass=${this.hass}
+            .value=${item.value?.toString()}
+            allow-custom-value
+            hide-clear-icon
+            .items=${Object.entries(item.metadata.states).map(
+              ([value, label]) => ({ value, label: `${value} - ${label}` })
+            )}
```

Comment:
Can we move items computation outside of the html rendering (line 330) so it's easier to read.